moonraker/components/power.py

```diff
@@ -684,11 +689,11 @@ async def _send_tplink_command(self,
                 'system': {'set_relay_state': {'state': int(command == "on")}}
             }
             # TPLink device controls multiple devices
-            if len(self.addr) == 2:
+            if self.output_id != 0:
```

Comment:
Is it possible for a smart strip to have an output id of 0?  If so, will it work without specifying the child id?

I'm wondering if we should default `output_id` to None and check for a None type rather than zero.  If the ID can't be zero, or if it defaults to 0, then we can leave it as it is.

Reply:
I agree that seems safer.